core/catalog-v001.xml

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
-    <uri id="Imports Wizard Entry" name="http://purl.obolibrary.org/obo/bfo/2020/bfo.owl" uri="http://purl.obolibrary.org/obo/bfo/2020/bfo.owl"/>
-    <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/" uri="./meta/AnnotationVocabulary.rdf"/>
-    <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/MappingCommonsToIOF/" uri="./commonstocoremapping/MappingCommonsToIOF.rdf"/>
-    <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/meta/MappingAnnotationVocabularyToCommons/" uri="./commonstocoremapping/meta/MappingAnnotationVocabularyToCommons.rdf"/>
-</catalog>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+    <uri id="Imports Wizard Entry" name="http://purl.obolibrary.org/obo/bfo/2020/bfo.owl" uri="http://purl.obolibrary.org/obo/bfo/2020/bfo.owl"/>

```

Comment:
Do we want to set up the cache before releasing?

Reply:
lets finalize in the architecture call on Thursday, my guess would be yes we want to set it up to avoid any further issues

Reply:
BFO was still broken yesterday, as far as I can tell; so yes, let’s cache BFO before release.

Reply:
We decided yesterday to do a separate pull request for caching, in case anything breaks.